main.go

```diff
@@ -1,32 +1,27 @@
 package main
 
 import (
-
 	"log"
-	"github.com/gorilla/mux"
 	"net/http"
+
+	"github.com/gorilla/mux"
+
 	"capi/api"
```

Comment:
I've split `"capi/api"` out to the bottom as I know it's a local import. 
Community import convention style seems to follow:

```
import (
	// stdlib Imports
	...

	// External Imports
	...

	// Internal Imports
	...
)